example/calc/calc.bnf

```diff
@@ -1,15 +1,14 @@
 /* Lexical part */
 
 _digit : '0'-'9' ;
-
-int64 : '1'-'9' {_digit} ;
+int64 : '1'-'9' {_digit} | '0' ;
```

Comment:
This surely handles cases like `1 + 0` as you've noted in the testcase, but fails when dealing with say `1 + 01` (which was also the case previously).
I guess a better fix for handling all sorts of whole number notations would be -
```
int64 : '0'-'9' {_digit} ;
```
This handles both kind of testcases described above. What do you think?